### DISCUSSION TITLE
Headers in output files

### DISCUSSION THREAD

**Discussion by baileythegreen** [widdowquinn/pyani#329] [Q&A]
Pursuant to the recent Issue #327, do (some of) our output files not have headers by design? If not, we might consider adding them. If they are omitted for a reason, we need to make sure the documentation makes it easy to find out what is in the output files. I would say this needs to be done even if it is only applicable for v2.

**Comment by widdowquinn** [comment]
Headers in output files are fine, but the BLAST .tab output in #327 is an intermediate file isn't it? As an intermediate file not intended for users, I don't think we have any obligation to provide the headers.

BLAST - by default - doesn't give headers for that format, and we don't need them. The intermediate files should not, in my view, be considered "`pyani` output".

**Reply by baileythegreen** [reply]
I ran a test locally and realised that there were both `.blast_tab` and `.blast_tab.dataframe` output files, and that the second did have a header. I had thought the user was looking at the final output file, and was surprised it didn't have anything.

That may mean there is nothing that needs to be done, here.

**Reply by widdowquinn** [reply]
I'd agree. The dataframe only has headers because that's the `pandas` default, IIRC.
